Dockerfile.unittest

```diff
@@ -0,0 +1,13 @@
+FROM python:3 as base
+WORKDIR app/
+
```

Comment:
I think we need a one line comment here describing what the test is doing / trying to test :)

Reply:
Hi @rosepearson I have made the changes but would like if you think I need to put more details on exactly what is being tested, since so far we do not have many unit tests but they will grow over time